docs/03-tutorials/00-application-connectivity/ac-09-get-api-specification.md

```diff
@@ -6,13 +6,13 @@ To view or download the API specification directly from this website, see the **
 
 ## Connector Service API
 
-To get the API specification for the Connector Service on a local Kyma deployment, run this command:
+To get the API specification for Connector Service on a local Kyma deployment, run this command:
 
 ```bash
-curl https://connector-service.kyma.local/v1/api.yaml
+curl https://connector-service.local.kyma.dev/v1/api.yaml
 ```
 
-Alternatively, get the API specification directly from the Connector Service: 
+Alternatively, get the API specification directly from Connector Service: 
```

Comment:
Remove this and replace `local.kyma.dev` with `$CLUSTER_DOMAIN` in the command above. And remove the mention of local.

Reply:
Fixed. The commands have been merged into one.